warehouse/integrations/datalake/datalake_test.go

```diff
@@ -407,4 +417,145 @@ func TestIntegration(t *testing.T) {
 		}
 		testhelper.VerifyConfigurationTest(t, dest)
 	})
+
+	t.Run("Trino", func(t *testing.T) {
+		dsn := fmt.Sprintf("http://user@localhost:%d?catalog=minio&schema=default&session_properties=minio.parquet_use_column_index=true",
+			trinoPort,
+		)
+		db, err := sql.Open("trino", dsn)
+		require.NoError(t, err)
+
+		require.Eventually(t, func() bool {
+			_, err := db.ExecContext(ctx, "SELECT 1")
+			return err == nil
+		}, 60*time.Second, 1*time.Second)
```

Comment:
We could try more often, we don't even have log pollution here. Every 100ms for example could work.